api/main.go

```diff
@@ -140,6 +140,7 @@ func main() {
 
 	// Ingest APIs: receive data from firewalls
 	authorized := router.Group("/ingest", middleware.BasicAuth())
+	authorized.POST("/info", methods.ForcePushInfo)
```

Comment:
Why naming Force?
```suggestion
	authorized.POST("/info", methods.PushInfo)
```

Reply:
Cause we override the info in the files :man_shrugging: 

Changed!

Reply:
Cause we override the info in the files 🤷‍♂️

Changed!